test_util/azure_test_driver.py

```diff
@@ -19,11 +19,22 @@
 from test_util.test_runner import integration_test
 
 
-def get_dcos_ui(master_url):
-    try:
-        return requests.get(master_url)
-    except requests.exceptions.ConnectionError:
-        pass
+def validate_env():
+    '''
+    The following environment variables must be set to ensure the template can be deployed successfully and that
+    parameters we rely on from a testing perspective are explicitly set and not relying on defaults which may change.
+    '''
+    required = ['AZURE_PARAM_linuxAdminUsername', 'AZURE_PARAM_oauthEnabled',
```

Comment:
This parameter is specific to the ACS templates, there is no equivalent with the azuredeploy templates. Essentially, I'm updating the test driver so it works with our main template (ACS) and dropping support for testing the other template type. We could continue to try and support testing the legacy template but that seems like a waste of effort right now.